### ISSUE TITLE
Linter suggests to add @EqualsAndHashCode to declare callSuper, but already overriden

### ISSUE THREAD

**Issue by bryant1410** [mplushnikov/lombok-intellij-plugin#184]
In IntelliJ (Android Studio, specifically), with `@Data` annotation and with the class inheriting from another, if `equals` and `hashCode` methods are overriden it anyway suggests to add `@EqualsAndHashCode` to declare `callSuper` attribute, but it shouldn't.

**Comment by mplushnikov** [comment]
I just corrected the intellij inspection for this usecase, it should be available in the next plugi verion >= 0.10.

```java
import lombok.Data;

public class DataWithOverrideEqualsAndHashCodeSugestion184 {
  @Data
  private static class SomeBasic {
    private int property;
  }

  @Data
  private static class SomeComplex extends SomeBasic {
    private final String value;

    @Override
    public boolean equals(Object o) {
      if (this == o) {
        return true;
      }
      if (o == null || getClass() != o.getClass()) {
        return false;
      }

      SomeComplex that = (SomeComplex) o;

      return value.equals(that.value);
    }

    @Override
    public int hashCode() {
      return value.hashCode();
    }
  }

  public static void main(String[] args) {
    SomeComplex complex = new SomeComplex("This is very complex");
    System.out.println(complex);
  }
}
```

But you will still get warning during compiling from lombok itself. 
![image](https://cloud.githubusercontent.com/assets/1204509/14043501/3524d620-f285-11e5-9333-a32dde89bb29.png)
This can only be fixed by lombok authors (@rzwitserloot @rspilker ).

**Comment by mplushnikov** [comment]
New version 0.10 of the plugin was released today and contains fixes and improvements for this issue.
